### DISCUSSION TITLE
Need help using Axiom with nuclei

### DISCUSSION THREAD

**Discussion by parvbarot** [pry0cc/axiom#727] [Q&A]
Hello everyone,

I'm relatively new to using Axiom and currently facing some challenges when trying to incorporate nuclei into my workflow. I was hoping to get some guidance and clarification on how to effectively utilize Axiom with nuclei.

Specifically, I'm looking for assistance in understanding the steps required to integrate nuclei into my Axiom project. I have reviewed the documentation and examples available, but I'm still encountering difficulties in implementing it correctly. It would be immensely helpful if someone could provide a step-by-step walkthrough or share their experiences in successfully combining Axiom and nuclei.

Here are a few questions I have:

What are the essential prerequisites and dependencies for using nuclei with Axiom?
How should I structure my Axiom project to incorporate nuclei effectively?
Are there any specific considerations or best practices I should keep in mind while using nuclei with Axiom?
Are there any known issues or limitations when working with nuclei and Axiom together?
I'm eager to learn and would greatly appreciate any insights, tips, or suggestions from the community. Thank you in advance for your support!

**Comment by pdelteil** [comment]
This read like a academic questionnaire/exam? 

Nuclei is already include in Axiom once you build an image using the default config file (check `.axiom/images/provisioners`)
